lib/src/room.dart

```diff
@@ -213,6 +213,12 @@ class Room {
   /// from the database, that you need to correctly calculate the displayname
   /// and the avatar of the room.
   Future<List<User>> loadHeroUsers() async {
+    // For invite rooms request own user and invitor.
+    if (membership == Membership.invite) {
+      final ownUser = await requestUser(client.userID!, requestProfile: false);
+      if (ownUser != null) await requestUser(ownUser.senderId);
```

Comment:
this looks like something requestUser already handles internally with requestProfile true?

Reply:
I don't see where 🤔 this assumes that the own m.room.member state event has a different senderId than the stateKey and the senderId is the inviting user. We want to fetch this user in any case for invites as this user is likely be necessary for calculating room name and avatar

Reply:
ah yes I have misread this then